### PR TITLE
ED-2183 send collaborators email

### PR DESCRIPTION
**Pull request by sdonk**

### CODE REVIEW

File: api/settings.py [hunk 1]
```diff
@@ -269,6 +269,8 @@
 DEFAULT_FROM_EMAIL = os.getenv("DEFAULT_FROM_EMAIL")
 FAS_FROM_EMAIL = os.getenv("FAS_FROM_EMAIL")
 FAB_FROM_EMAIL = os.getenv("FAB_FROM_EMAIL")
+FAB_OWNERSHIP_LINK = os.getenv("FAB_OWNERSHIP_LINK")
```
Comment (richtier) [inline api/settings.py]: orhter env vars use `_URL`, e.g., `FAB_FROM_URL`

File: company/models.py [hunk 2]
```diff
@@ -304,6 +304,20 @@ class OwnershipInvite(TimeStampedModel):
     accepted = models.BooleanField(default=False)
     accepted_date = models.DateTimeField(null=True, blank=True)
 
+    @property
+    def subject(self):
+        return 'Accept account ownership invitation'
```
Comment (richtier) [inline company/models.py]: this should be a setting.

could be
```
class OwnershipInvite:
    email_subject = settings.OWNERSHIP_INVITE_SUJECT
```

File: company/signals.py [hunk 3]
```diff
@@ -43,3 +46,22 @@ def delete_company_elasticsearch_document(sender, instance, *args, **kwargs):
...
+):
+    from notifications.tasks import send_email
+
+    if not created:
+        return
+
+    send_email.delay(
+        subject=instance.subject,
+        text_body=instance.invite_link,
+        html_body='<html><a href="{}">Click</a></html>'.format(
```
Comment (kowalcj0) [inline company/signals.py]: @sdonk We're not going to send anything? just the invite link itself?

Comment (sdonk) [reply company/signals.py]: It's placeholder content atm

**Pull request merged**